packages/node_modules/@ciscospark/react-redux-spark-fixtures/src/__mocks__/spark-store.js

```diff
@@ -9,8 +9,8 @@ import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 
 import {initialState as mercury} from '@ciscospark/redux-module-mercury';
+import reducers, {initialState} from '@ciscospark/react-redux-spark';
```

Comment:
Do we want to take this opportunity to rename to `react-redux-webex` or keep it atomic?

Reply:
We would need to do a deprecation on npm since this is actually one of the packages published to npm, so I would say no at this time.